base/essentials.jl

```diff
@@ -765,3 +765,7 @@ Indicate whether `x` is [`missing`](@ref).
 """
 ismissing(::Any) = false
 ismissing(::Missing) = true
+
+function popfirst! end
```

Comment:
Do we still need this line?

Reply:
yes, this generally gets loaded quite early.